google/cloud/compute/v1/compute_gapic.yaml

```diff
@@ -18,15 +18,4 @@ config_schema_version: 2.0.0
 language_settings:
   java:
     package_name: com.google.cloud.compute.v1
-  python:
-    package_name: google.cloud.compute_v1.gapic
-  go:
-    package_name: cloud.google.com/go/compute/apiv1
-  csharp:
-    package_name: Google.Compute.V1
-  ruby:
-    package_name: Google::Cloud::Compute::V1
-  php:
```

Comment:
Are these ok to remove? Just making sure this is intentional

Reply:
Yes - only the Java and PHP microgenerators will ever use the `long_running` and `batching` options in this file now. This file can be deleted once Java migrates as well.